openapi/kas-fleet-manager.yaml

```diff
@@ -1413,8 +1431,8 @@ components:
         values:
           type: array
           items:
-            $ref: '#/components/schemas/values'
-    values:
+            $ref: '#/components/schemas/Values'
+    Values:
```

Comment:
does changing the case of these values create any /much chaos in the repo?

Reply:
It should be fine. But let's run `make openapi/generate code/fix` and push the changes to see how far CI cries :-D